_gsocprojects/2025/project_HSFCondDB.md

```diff
@@ -0,0 +1,20 @@
+---
+project: HSFCondDB
+title: HSF Conditions Database 
+layout: default
+description: |
+  In high-energy physics (HEP), conditions databases play a critical role in managing non-event data. 
+  This includes calibration constants, alignment parameters, and detector conditions, which evolve over time. 
+  These databases ensure that analysis software can access the correct calibration and alignment data corresponding to 
+  the detector’s state at any given time, enabling accurate physics measurements.
+  
+  The [HEP Software Foundation](https://hepsoftwarefoundation.org/) (HSF) proposes a Conditions Database reference 
+  for HEP and Nuclear Physics experiments around the world. Several experts have converged on a common design for 
+  conditions data access management [arXiv:1901.05429](https://arxiv.org/abs/1901.05429). This reference implementation 
+  has been successfully operating within the [sPHENIX](https://www.sphenix.bnl.gov/) experiment for nearly two years 
+  and is currently being adopted by [Belle II](https://www.belle2.org/). Additionally, other collaborations, including 
+  [ePIC](https://www.bnl.gov/eic/epic.php) and the [Einstein Telescope](https://www.et-gw.eu/), have expressed interest 
+  in evaluating its suitability for their needs.
```

Comment:
Nitpick, could you add a one sentence `summary` like here: https://github.com/HSF/hsf.github.io/blob/main/_gsocprojects/2025/project_Constellation.md?plain=1#L13 ?

Otherwise the description blows up the formatting of the main webpage